packages/stream_video/lib/src/call/call.dart

```diff
@@ -1979,6 +1980,16 @@ class Call {
     return result.map((_) => none);
   }
 
+  Future<Result<bool>> setMultitaskingCameraAccessEnabled(bool enabled) async {
+    if (CurrentPlatform.isIos) {
+      final result =
+          await rtc.Helper.enableIOSMultitaskingCameraAccess(enabled);
+      return Result.success(result);
+    }
+
+    return const Result.success(false);
+  }
+
```

Comment:
Same problem with this one as before. If it is a "set" method, let's add a named parameter "enable".